File: KahveQRKodOlusturucuJetpack-calismalarim/app/src/main/java/com/example/qrkodolusturucujetpack/services/Services.kt
package com.example.qrkodolusturucu.services


import com.example.qrkodolusturucu.model.CRUDCevap
import com.example.qrkodolusturucujetpack.model.KodUretCevap
import com.example.qrkodolusturucujetpack.model.UrunCevap
import com.example.qrkodolusturucujetpack.model.UrunKategori

interface Services {
    suspend fun kullaniciEkle(kisiTel:String): CRUDCevap
    suspend fun tumKahveler(): UrunCevap
    suspend fun kodUret(dogrulamaKodu:String,kisiTel:String): KodUretCevap
    suspend fun kahveWithKategoriId(urunKategori: UrunKategori): UrunCevap
    suspend fun kodSil(dogrulamaKodu: String): KodUretCevap
}

File: kahve_qr_olusturucu_flutter-calismalarim/lib/services/Services.dart


import '../model/CRUDCevap.dart';
import '../model/KodUretCevap.dart';
import '../model/UrunCevap.dart';
import '../model/UrunKategori.dart';

abstract class Services {
  Future<CRUDCevap> kullaniciEkle(String kisiTel);
  Future<UrunCevap> tumKahveler();
  Future<KodUretCevap> kodUret(String dogrulamaKodu, String kisiTel);
  Future<UrunCevap> kahveWithKategoriId(UrunKategori urunKategori);
  Future<KodUretCevap> kodSil(String dogrulamaKodu);
}
